<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifierEtSupprimerDialog</class>
 <widget class="QDialog" name="ModifierEtSupprimerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modifier ou supprimer une ressource</string>
  </property>
  <widget class="QLabel" name="idLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>idRessource:</string>
   </property>
  </widget>
  <widget class="QLabel" name="idAfficherLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="nomLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nom:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nomLineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>xxx</string>
   </property>
  </widget>
  <widget class="QLabel" name="prenomLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Prenom: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="prenomLlineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>xxx</string>
   </property>
  </widget>
  <widget class="QLabel" name="typeLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Type:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="typeComboBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>300</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="modifierPushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
  </widget>
  <widget class="QPushButton" name="supprimerPushButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>400</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="annulerPushButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>400</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
